### PR TITLE
Move linkedin url code from bz-script div to module JS

### PR DESCRIPTION
**Pull request by rshipp**
Task: https://app.asana.com/0/1170776727341290/1173524622792756/f

Screenshot:

Screen Shot 2020-05-18 at 11 50 47 AM

### CODE REVIEW

File: braven_custom.js [hunk 1]
```diff
@@ -1129,6 +1129,13 @@ function bzAjaxLoad() {
       jQuery(this).addClass('bz-ajax-loaded-linkedin bz-ajax-loaded');
     }
   });
+  // Fill out the text input below the button with the user's LinkedIn URL.
+  jQuery.get('/bz/user_linkedin_url?' + Math.random()).success(
+    function(ajaxResponse) {
+      var evt = document.createEvent('HTMLEvents');
+      evt.initEvent('change', true, true);
+      jQuery('.bz-user-linkedin-profile').val(ajaxResponse.linkedin_url)[0].dispatchEvent(evt);
+    });
```
Comment (jessouyangbraven) [inline braven_custom.js:1138]: Weird tab?

Comment (rshipp) [reply braven_custom.js:1138]: we do this super inconsistently in the js, I'm not sure which is better. should I change it to this?

```js
    }
  );
```

Comment (rshipp) [reply braven_custom.js:1138]: ohh, or maybe 

```js
  jQuery.get('/bz/user_linkedin_url?' + Math.random()).success(function(ajaxResponse) {
    ...
  });
```

Comment (jessouyangbraven) [reply braven_custom.js:1138]: ...it is super inconsistent. Ignore me, then, and you pick.

**Pull request merged**